tools/validators/ontology_validator/yamlformat/tests/presubmit_validate_types_lib_test.py

```diff
@@ -246,7 +249,7 @@ def testConfigUniverseGetUnitsForMeasurementMultipleNoUnits(self):
         unit_universe=unit_universe)
 
     units = config_universe.GetUnitsForMeasurement('powerfactor_sensor')
-    self.assertSameElements(['no_units'], units)
```

Comment:
can you please leave an empty line here, and remove the empty line above, 
we do follow this approach: https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/

we use blank lines to separate the parts.
//Arrange
//blank line
//Act
//blank line
//assert